tgui/packages/tgui/components/Section.tsx

```diff
@@ -103,9 +103,9 @@ export const Section = forwardRef(
         {...computeBoxProps(rest)}
       >
         {hasTitle && (
-          <div className="Section__title">
-            <span className="Section__titleText">{title}</span>
-            <div className="Section__buttons">{buttons}</div>
+          <div className="Section__head">
+            <div className="Section__headTitle">{title}</div>
+            <div className="Section__headButtons">{buttons}</div>
```

Comment:
you'd need to make a commit to [https://github.com/tgstation/tgui-core]( https://github.com/tgstation/tgui-core) as these are being removed. not sure why this change is here either way

Reply:
idk bro i didnt make the ui

Reply:
npnp